hangman/lib/hangman.ex

```diff
@@ -1,9 +1,9 @@
 defmodule Hangman do
```

Comment:
# -10

So the general structure here is a little wonky.

In class we had the start-link and the API functions that call GenServer.call/cast in the top-level module.

The server module then has the handle_call functions, which then call Game....